### PR TITLE
Proposing Istio to become a CNCF project

### PR DESCRIPTION
**Pull request by craigbox**
On behalf of the Istio Steering Committee, please find attached a proposal for Istio to become a CNCF project.

### What is Istio?

[Istio](https://istio.io) is an open source service mesh that provides a uniform, efficient and transparent way to secure, connect, and monitor services in cloud native applications. It supports zero-trust networking, policy enforcement, traffic management, load balancing, and monitoring; all without requiring applications to be rewritten.

Istio extends Kubernetes to establish a configurable, application-aware network leveraging the Envoy service proxy. It can manage cloud native and traditional workloads alike, supporting from single-cluster to complex multi-network deployments.

### Why are we proposing to join the CNCF?

Istio is taking this step as a mature project, approaching its 5th birthday. At this stage, it has reached a maturity where it can be a major driver for adoption of the cloud native model and ecosystem more broadly.

With the recent acceptance of Knative as an incubating project to the CNCF, we feel that it is the ideal time to join a vendor-neutral home for further community-driven innovation. This includes the ownership of the Istio trademark, which will be transferred to the Linux Foundation. The trademark will remain available for the community to use under the Open Usage Commons' [published usage guidelines](https://openusage.org/trademark-guidelines/).

Istio stands on the shoulders of several CNCF projects, such as Kubernetes, Envoy, gRPC, Prometheus and SPIFFE, and many contributors in the Istio community are actively involved with these projects as well. It can be installed with Helm, and integrations exist for many other CNCF projects including Knative, Flagger, Jaeger, Open Policy Agent, and OpenTelemetry.  We think that bringing Istio to the CNCF will help accelerate the feedback, feature, and development cycle of not just Istio, but also its partner projects.

Please see the project proposal document for further details.

### CODE REVIEW

**Comment by dims** [comment]
@craigbox hurray! however the markdown file is empty :)

**Comment by sdake** [comment]
@craigbox nicely done!

You mentioned transfer of the trademark to LF but also availability under OUC.

Would you elaborate on this concept?

File: proposals/incubation/istio.md [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+* Apache 2.0
+* BSD License
+* ISC License
+* MIT License
+
+Five modules are licensed under the Mozilla Public License, and all have been [granted exceptions](https://github.com/cncf/foundation/blob/main/license-exceptions/cncf-exceptions-2019-11-01.spdx) by the CNCF governing board.
+
+Licenses for most Istio dependencies are automatically included at https://github.com/istio/istio/tree/master/licenses. A snapshot of modules and their corresponding licenses [can be found below](#licenses).
+
+Projects in istio-ecosystem will be brought in line with CNCF guidelines before submission.
```
Comment (sdake) [inline proposals/incubation/istio.md:46]: Do these subprojects not have a choice? Presuming they have a choice, isn’t it odd that integrated projects could become sandbox projects without a separate application and oversight?

File: proposals/incubation/istio.md [hunk 2]
```diff
@@ -0,0 +1,375 @@
...
+
+### Issue tracker
+
+https://github.com/istio/istio/issues
+
+### Committers
+
+Commits to Istio, no matter how small, are welcome from anyone in the community. 
+
+Istio uses "maintainer" as the noun for a member allowed to approve code contributions. See https://github.com/istio/community/blob/master/ROLES.md#maintainer for more information.
```
Comment (sdake) [inline proposals/incubation/istio.md:56]: 👍

File: proposals/incubation/istio.md [hunk 3]
```diff
@@ -0,0 +1,375 @@
...
+### External dependencies
+
+Istio has external dependencies licensed under the following CNCF-approved licenses:
+
+* Apache 2.0
+* BSD License
+* ISC License
+* MIT License
+
+Five modules are licensed under the Mozilla Public License, and all have been [granted exceptions](https://github.com/cncf/foundation/blob/main/license-exceptions/cncf-exceptions-2019-11-01.spdx) by the CNCF governing board.
```
Comment (sdake) [inline proposals/incubation/istio.md:42]: When analyzing licenses prior, there were some GPL transitive dependencies. Are those present in existing Istio?

File: proposals/incubation/istio.md [hunk 4]
```diff
@@ -0,0 +1,375 @@
...
+
+Istio has had over [8000 individual contributors](https://istio.teststats.cncf.io/d/18/overall-project-statistics-table?orgId=1) on the project’s GitHub repositories. 
+
+The project has over [32,000 GitHub stars](https://istio.teststats.cncf.io/d/18/overall-project-statistics-table?orgId=1) across its repositories.
+
+[Over 70 companies](https://istio.teststats.cncf.io/d/5/companies-table?orgId=1&var-period_name=Last%20decade&var-metric=prs) have submitted at least 8 Pull Requests to Istio.
+
+#### Contribution
+
+Istio's proud parents are Google and IBM, alongside the Lyft team who built the Envoy proxy.
```
Comment (sdake) [inline proposals/incubation/istio.md:124]: Cisco was a parent as well. The use of the word proud is odd.

File: proposals/incubation/istio.md [hunk 5]
```diff
@@ -0,0 +1,375 @@
...
+### Community size
+
+Istio has had over [8000 individual contributors](https://istio.teststats.cncf.io/d/18/overall-project-statistics-table?orgId=1) on the project’s GitHub repositories. 
+
+The project has over [32,000 GitHub stars](https://istio.teststats.cncf.io/d/18/overall-project-statistics-table?orgId=1) across its repositories.
+
+[Over 70 companies](https://istio.teststats.cncf.io/d/5/companies-table?orgId=1&var-period_name=Last%20decade&var-metric=prs) have submitted at least 8 Pull Requests to Istio.
+
+#### Contribution
+
```
Comment (sdake) [inline proposals/incubation/istio.md:123]: This section could focus on individuals instead of firms or in addition…

**Review by sdake — COMMENTED** [review]
Looks pretty good - like it was written as Istio release notes were (as a team). Few comments that I feel are important to address or explain.

File: proposals/incubation/istio.md (hunk 4 above)
Comment (craigbox) [reply proposals/incubation/istio.md:124]: "proud parents" is a simply figure of speech in English.  Happy to remove it if it does not translate well.

Comment (sdake) [reply proposals/incubation/istio.md:124]: Right - thanks for the clarity. I would recommend avoiding idiomatic expressions.

File: proposals/incubation/istio.md [hunk 6]
```diff
@@ -0,0 +1,375 @@
...
+* Fine-grained control of traffic behavior with rich routing rules, retries, failovers, and fault injection
+* A policy API supporting access controls, rate limits and quotas, that can be connected to external authorization systems
+* Automatic generation of metrics, logs, and traces for all traffic within a cluster, including cluster ingress and egress
+* A data plane platform allowing for extension and customization of proxy behavior
+
+Istio is designed for extensibility, and can handle a diverse range of deployment needs. Istio’s control plane runs on Kubernetes, and is capable of managing workloads running in the same Kubernetes cluster, other Kubernetes clusters, and even workloads running outside Kubernetes on VMs.
+
+### Sponsor / Advisor from TOC
+
+To be appointed.
```
Comment (dims) [inline proposals/incubation/istio.md:22]: Thanks for your application and welcome to CNCF.

the TOC team will figure out who can help with the incubation process and let you all know

Comment (dzolotusky) [reply proposals/incubation/istio.md:22]: @craigbox I'll be your sponsor for this process.
Let's connect next week to figure out next steps and get this moving.

**Comment by dims** [comment]
Thanks @dzolotusky !! assigned to you.

File: proposals/incubation/istio.md (hunk 6 above)
Comment (thisisnotapril) [reply proposals/incubation/istio.md:22]: Thank you Dave for the sponsorship! Steering is looking forward to working with you on this. Please use istio-steering@googlegroups.com  for comms so we're able to track all the moving parts!

Comment (erinaboyd) [reply proposals/incubation/istio.md:22]: I can also co-sponsor if you need help @dzolotusky :)

File: proposals/incubation/istio.md (hunk 3 above)
Comment (craigbox) [reply proposals/incubation/istio.md:42]: Not according to our current analysis.  The CNCF legal team will inform us if they disagree!

File: proposals/incubation/istio.md (hunk 1 above)
Comment (craigbox) [reply proposals/incubation/istio.md:46]: There are a types of project in istio-ecosystem: 

* contributed projects (e.g. Admiral)
* useful samples, maintained by the Istio project (e.g. wasm-extensions)
* things that are wildly out of date, which will be archived. 

For the first two, we will either help the teams move to their own orgs, or if they wish to remain in the "Istio incubator" (similar to the kubernetes-sigs repository, where projects exist but are not granted the same status as the parent project), they will individually need to be updated to meet CNCF standards.

File: proposals/incubation/istio.md (hunk 6 above)
Comment (craigbox) [inline proposals/incubation/istio.md:22]:
```suggestion
Dave Zolotusky, with help from Erin Boyd (TOC) and Lee Calcote (TAG Network)
```

**Comment by craigbox** [comment]
https://www.cncf.io/blog/2022/09/28/istio-sails-into-the-cloud-native-computing-foundation/

**Comment by justaugustus** [comment]
> https://www.cncf.io/blog/2022/09/28/istio-sails-into-the-cloud-native-computing-foundation/

Very exciting! Congrats everyone!!!

**Pull request merged**